plugins/arDominionB5Plugin/modules/repository/templates/_uploadLimit.php

```diff
@@ -67,7 +71,10 @@ class="modal fade"
 
           <form id="upload-limit-form" method="POST" action="<?php echo url_for([$resource, 'module' => 'repository', 'action' => 'editUploadLimit']); ?>">
             <?php echo $form->renderHiddenFields(); ?>
-            <div>
+	    <div>
+	      <style <?php echo __(sfConfig::get('csp_nonce', '')); ?>>
+                #uploadLimit_value { width: 6em }
```

Comment:
replace with BS5 if possible. Unsure if BS5 has width values for 'em' units?

Reply:
BS doesn't have specfic classes non percentage sizing em so will leave as is.